app/views/parents/SidebarComponent.vue

```diff
@@ -76,6 +76,38 @@
             <span class="sidebar__tabs__name">Online Classes</span>
           </li>
         </router-link>
+        <router-link
+          tag="a"
+          to="/roblox"
+          target="_blank"
+        >
+          <li
+            :class="{ sidebar__tabs__item: true, sidebar__tabs__item__sel: selectedTab === 'roblox' }"
+          >
+            <img
+              src="/images/ozaria/teachers/dashboard/svg_icons/IconComputer.svg"
+              alt="Roblox"
+              class="sidebar__tabs__img"
+            >
+            <span class="sidebar__tabs__name">Roblox</span>
+          </li>
+        </router-link>
+        <router-link
+          tag="a"
+          to="/ai"
+          target="_blank"
+        >
+          <li
+            :class="{ sidebar__tabs__item: true, sidebar__tabs__item__sel: selectedTab === 'ai-hackstack' }"
+          >
+            <img
+              src="/images/ozaria/teachers/dashboard/svg_icons/Icon_Progress_Black.svg"
+              alt="AI Hackstack"
+              class="sidebar__tabs__img"
+            >
+            <span class="sidebar__tabs__name">AI Hackstack Beta</span>
```

Comment:
beta appears differently in teacher dashboard. Let's replicate it here
<img width="325" alt="Screenshot 2024-07-05 at 9 25 21 AM" src="https://github.com/codecombat/codecombat/assets/6376157/1625af3c-8152-4319-b176-02f8e9adaa33">